### ISSUE TITLE
(20–25 sec) Camera rotates slightly around the boy, room lights glowing softly (25–30 sec) Final shot – logo fully completed on screen with bright glow and sparkle effect

### ISSUE THREAD

**Issue by nansabhaibhai591-hub** [SkyworkAI/SkyReels-V2#382]
"3D cartoon style young boy sitting at a study desk, happily designing a colorful logo on his laptop. The character has big expressive eyes, smooth animated movements, and a friendly smile (Pixar-style). Soft warm lighting with glowing fairy lights in the background, cozy creative room setup. Gentle background music playing, synced with smooth animations.

**Comment by nansabhaibhai591-hub** [comment]
"3D cartoon style young boy sitting at a study desk, happily designing a colorful logo on his laptop. The character has big expressive eyes, smooth animated movements, and a friendly smile (Pixar-style). Soft warm lighting with glowing fairy lights in the background, cozy creative room setup. Gentle background music playing, synced with smooth animations.

**Comment by nansabhaibhai591-hub** [comment]
Scene progression: (0–5 sec) Wide shot – boy opens laptop and starts working, slight head movement, curious expression
(5–10 sec) Medium shot – boy moving mouse, colorful shapes appearing on screen like animation magic

**Comment by speedrecorddevil123-stack** [comment]
import torch
from diffusers import AutoModel, SkyReelsV2DiffusionForcingPipeline, UniPCMultistepScheduler
from diffusers.utils import export_to_video

vae = AutoModel.from_pretrained("Skywork/SkyReels-V2-DF-14B-540P-Diffusers", subfolder="vae", torch_dtype=torch.float32)

pipeline = SkyReelsV2DiffusionForcingPipeline.from_pretrained(
    "Skywork/SkyReels-V2-DF-14B-540P-Diffusers",
    vae=vae,
    torch_dtype=torch.bfloat16
)
flow_shift = 8.0  # 8.0 for T2V, 5.0 for I2V
pipeline.scheduler = UniPCMultistepScheduler.from_config(pipeline.scheduler.config, flow_shift=flow_shift)
pipeline = pipeline.to("cuda")

prompt = "A cat and a dog baking a cake together in a kitchen. The cat is carefully measuring flour, while the dog is stirring the batter with a wooden spoon. The kitchen is cozy, with sunlight streaming through the window."

output = pipeline(
    prompt=prompt,
    num_inference_steps=30,
    height=544,  # 720 for 720P
    width=960,   # 1280 for 720P
    num_frames=97,
    base_num_frames=97,  # 121 for 720P
    ar_step=5,  # Controls asynchronous inference (0 for synchronous mode)
    causal_block_size=5,  # Number of frames in each block for asynchronous processing
    overlap_history=None,  # Number of frames to overlap for smooth transitions in long videos; 17 for long video generations
    addnoise_condition=20,  # Improves consistency in long video generation
).frames[0]
export_to_video(output, "T2V.mp4", fps=24, quality=8)
